### PR TITLE
✨ Trim default startup output and add --verbose to dev and run

### PR DESCRIPTION
**Pull request by patrick91**
CleanShot 2026-07-09 at 14 57 19@2x

## Stack [🍰](https://shortcake.patrick.wtf)

- #458 (`2026-07-03-use-minimal-startup-output-outside-tty`)
- #463 (`2026-07-07-bring-the-full-fastapi-style-for-sharing-with-fast`)
- **#471** (`2026-07-09-trim-startup-output-add-verbose`) <-- this PR
- #470 (merged) (`2026-07-09-redesign-startup-output-style`)
- #457 (merged) (`2026-07-03-avoid-fancy-logs-for-non-tty-output`)

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -40,20 +41,16 @@ def test_dev() -> None:
...
+        assert "Using import string: single_file_app:app" in result.output
         assert "Server started at http://127.0.0.1:8000" in result.output
         assert "Documentation at http://127.0.0.1:8000/docs" in result.output
-        assert (
-            "Running in development mode, for production use: fastapi run"
-            in result.output
-        )
-
-        assert "🐍 single_file_app.py" in result.output
+        # a passed path is explicit, so no source hint and no pyproject nudge
```
Comment (tiangolo) [inline tests/test_cli.py:48]: I think some people might not be aware that there's now a `pyproject.toml` config, I think we could hint also when there's an explicit path too so they can use that more convenient new way.

I think it's good we don't hint when it's already set, though. 🤓

**Review by tiangolo — APPROVED** [review]
Excellent! :tada: 

Edit:

~Just one small ask.~ after discussion internally, just ship it! :rocket:

File: tests/test_cli.py (hunk 1 above)
Comment (tiangolo) [reply tests/test_cli.py:48]: After discussing internally, this could get into edge cases that become hard to handle and we could misguide users, so let's keep it as you have it. :rocket:
